src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -109,11 +127,32 @@ public void InitializeService(ServiceHost serviceHost)
                     null,
                     null);
                 }
+            }
+
+            return dataContainer;
+        }
+
+        /// <summary>
+        /// Handles a backup request
+        /// </summary>
+        internal static async Task HandleBackupRequest(
```

Comment:
Per discussion, I created a new work item to work on the unit tests - https://github.com/Microsoft/carbon/issues/910